js/views/userPage/UserPageHome.js

```diff
@@ -0,0 +1,77 @@
+import loadTemplate from '../../utils/loadTemplate';
+import { View } from 'backbone';
+import app from '../../app';
+
+export default class extends View {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+
+    this.ownPage = options.ownPage;
+    this.isModerator = true; // TODO set if this user is a moderator
+    this.currentModerator = false; // TODO set if this user is one of the viewer's moderators
+  }
+
+  className() {
+    return 'userPageHome';
+  }
+
+  events() {
+    return {
+      'click .js-termsLink': 'termsClick',
+      'click .js-termsClose': 'termsClose',
+      'click .js-addModerator': 'addModeratorClick',
+    };
+  }
+
+  termsClick() {
+    this.$termsDisplay.toggleClass('open');
+    if (this.$termsDisplay.hasClass('open')) this.$termsDisplay[0].scrollIntoViewIfNeeded();
```

Comment:
+1 `scrollIntoViewIfNeeded` is a really cool method the browser folks have given us.